example/cr-detailed.yaml

```diff
@@ -32,6 +32,12 @@ spec:
     # see https://bookkeeper.apache.org/docs/latest/admin/autorecovery/
     autoRecovery: true
 
+    # To enable bookkeeper metrics feature
```

Comment:
Unrelated to this PR, I noticed that this file does not have a license header.

Reply:
This file is just a sample CR definition that can be taken and modified by anyone and it does not contain any intellectual property.